src/Raven.Server/Documents/Indexes/Static/DynamicArray.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Raven.Server.Documents.Indexes.Static
 {
-    public class DynamicArray : DynamicObject, IEnumerable<object>
+    public class DynamicArray : DynamicObject, IOrderedEnumerable<object>
```

Comment:
Do we need this?

Reply:
Needed for ThenBy

Reply:
Can't do `Enumerable.ThenBy(this, ...)` without it.
It expects `IOrderedEnumerable`.